torch/onnx/symbolic_opset12.py

```diff
@@ -209,3 +209,48 @@ def unfold(g, input, dimension, size, step):
         return squeeze
     else:
         return _unimplemented("Unfold", "input size not accessible")
+
+@parse_args('v', 'v', 'is', 'is', 'v')
+def tensordot(g, input_a, input_b, dims_a, dims_b, out=None):
+    if out is not None:
+        _unimplemented("Tensordot", "Out parameter is not supported for tensordot.")
+
+    from torch.onnx.symbolic_opset9 import view, permute
+    for i in range(len(dims_a)):
+        if sym_help._get_tensor_dim_size(input_a, dims_a[i]) != sym_help._get_tensor_dim_size(input_b, dims_b[i]):
```

Comment:
If both returns `None`, shall we also raise exception?

Reply:
Good catch.

Reply:
Has removed the check for the size here. Because sym_help._get_tensor_dim_size() will probably return a None in a reasonable scenario, like in my newly added test which uses dynamic_axes.

In that case, the size of those dimensions could not be got by calling sym_help._get_tensor_dim_size() method, but with my latest changes, we don't need to get the size values so None will has no impact to the function at all.